tfac/dataHelpers.py

```diff
@@ -72,15 +44,46 @@ def import_LINCS_CCLE():
     df.drop('Control', inplace=True, level=0)
     df = df.sort_index()
 
-    dfArray = df.to_numpy()
-    tensor = np.reshape(dfArray, (-1, len(times), dfArray.shape[1]))
+    # subtract the control
+    row1 = df.iloc[[0]].values[0] #control row
+    df = df.apply(lambda row: row - row1, axis=1)
```

Comment:
I think I asked you before whether we need this or not, if I remember correctly you said since these are protein levels it makes sense to first subtract the control.. I think scoring is to make the variance between proteins and RNAseq data as the same scale.

Reply:
The defaults for `scale` should both mean center _and_ variance scale. Either you want to only subtract off the control, in which case you don't want to mean center after, or you want to fully z-score, in which case subtracting off the control has no effect.

Reply:
Thank you, I fixed it.